jormungandr-lib/src/interfaces/fragment_log.rs

```diff
@@ -87,6 +87,10 @@ impl FragmentLog {
     /// set the new status
     #[inline]
     pub fn modify(&mut self, new_status: FragmentStatus) {
+        // we must not be able to transition from InABlock to Pending or Rejected
+        if self.status.is_in_a_block() && !new_status.is_in_a_block() {
+            return;
```

Comment:
Um, just that? No error, no warning message?

Reply:
Not really sure if this is worth even a warning level message. This is a normal scenario in the multi-leader operation mode.